pkg/crc/preflight/preflight_checks_darwin.go

```diff
@@ -182,3 +185,27 @@ func addFileWritePermissionToUser(filename string) error {
 
 	return nil
 }
+
+func removeCRCHyperkitMachine() error {
+	err := os.RemoveAll(filepath.Join(constants.MachineInstanceDir, "crc"))
+	if err != nil {
+		logging.Debugf("Failed to remove hyperkit machine directory: %v", err)
+		return err
+	}
+	return nil
+}
+
+func stopCRCHyperkitProcess() error {
+	path, err := exec.LookPath("pkill")
+	if err != nil {
+		logging.Debugf("Could not find 'pkill'. %v", err)
+		return fmt.Errorf("Could not find 'pkill'. %v", err)
```

Comment:
``` 
return errors.Wrap(err, "Could not find 'pkill'")
```

Reply:
Or `fmt.Errorf("Could not find 'pkill'. %w", err)` ?